pkg/kubelet/prober/prober_test.go

```diff
@@ -73,13 +92,23 @@ func TestGetURLParts(t *testing.T) {
 				},
 			},
 		}
-		p, err := extractPort(test.probe.Port, container)
+
+		scheme := test.probe.Scheme
+		if scheme == "" {
```

Comment:
it's not unfortunate, it's very deliberate.  Default values can vary by API version